client/extensions/woocommerce/lib/countries/index.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import CA from './CA';
+import US from './US';
+
+// Note: We are not using the other state name resources in calypso
```

Comment:
Does this comment refer to both the Calypso core data and the WooCommerce specific selectors that return this data?

https://github.com/Automattic/wp-calypso/blob/master/client/extensions/woocommerce/state/sites/locations/selectors.js

It will definitely be handy to use these (which the settings pages are also using) in the future, but for the sake of the deadline/v1 and to keep things moving, I would be fine using the hard-coded list and just adding a `TODO` here about this.